service/controller/v24/resource/cloudformation/delete.go

```diff
@@ -100,6 +108,47 @@ func (r *Resource) ApplyDeleteChange(ctx context.Context, obj, deleteChange inte
 	return nil
 }
 
+func disableMasterTerminationProtection(ctx context.Context, masterInstanceName string) error {
+	cc, err := controllercontext.FromContext(ctx)
```

Comment:
Can we have start and end logs in here? Basically `disabling master instance termination protection` and `disabled master instance termination protection`.